src/analysis/network/qgsvectorlayerdirector.h

```diff
@@ -74,12 +74,13 @@ class ANALYSIS_EXPORT QgsVectorLayerDirector : public QgsGraphDirector
      */
     void makeGraph( QgsGraphBuilderInterface *builder,
                     const QVector< QgsPointXY > &additionalPoints,
-                    QVector< QgsPointXY> &snappedPoints SIP_OUT ) const override;
+                    QVector< QgsPointXY> &snappedPoints SIP_OUT,
+                    QgsFeedback *feedback ) const override;
```

Comment:
Since you are checking if feedback is NULL internally, do you want to provide a default value for it here?